dotcom-rendering/src/components/RichLinkComponent.importable.tsx

```diff
@@ -79,68 +78,72 @@ export const RichLinkComponent = ({
+				width: '',
+				height: '',
+				source: '',
+				displayCredit: '',
+				photographer: '',
+				isMaster: '',
+				credit: '',
+				mediaId: '',
+			},
+			index: -1,
+			mediaType: '',
+			url: '',
+		},
+		contentType: 'article',
+		format: {
+			design: 'ArticleDesign',
+			display: 'StandardDisplay',
+			// We default to SpecialReport here purely because the greys of this theme
+			// look better as the defaults
+			theme: 'SpecialReportTheme',
+		},
+		cardStyle: 'news',
+		tags: [],
+		sponsorName: '',
+	} satisfies FERichLinkType;
+
 	const url = buildUrl(element, ajaxUrl);
-	const { data, error } = useApi<FERichLinkType>(url);
+	const { data, error } = useApi<FERichLinkType>(url, { fallbackData });
 
 	if (error) {
 		// Send the error to Sentry
 		window.guardian.modules.sentry.reportError(error, 'rich-link');
 	}
 
-	if (data?.imageAsset) {
-		const richLinkImageData: RichLinkImageData = {
-			thumbnailUrl: data.thumbnailUrl,
-			altText: data.imageAsset.fields.altText,
-			width: data.imageAsset.fields.width,
-			height: data.imageAsset.fields.height,
-		};
+	if (!data) return null;
 
-		return (
-			// This is the enhanced rich link rendered on the client
-			<RichLink
-				richLinkIndex={richLinkIndex}
-				cardStyle={data.cardStyle}
-				imageData={richLinkImageData}
-				headlineText={data.headline}
-				contentType={data.contentType}
-				url={data.url}
-				starRating={data.starRating}
-				linkFormat={decideFormat(data.format)}
-				format={format}
-				tags={data.tags}
-				sponsorName={data.sponsorName}
-				contributorImage={data.contributorImage}
-			/>
-		);
-	}
-
-	const defaultFormat = {
-		display: ArticleDisplay.Standard,
-		design: ArticleDesign.Standard,
-		// We default to SpecialReport here purely because the greys of this theme
-		// look better as the defaults
-		theme: ArticleSpecial.SpecialReport,
+	const richLinkImageData: RichLinkImageData = {
+		thumbnailUrl: data.thumbnailUrl,
+		altText: (data.imageAsset ?? fallbackData.imageAsset).fields.altText,
+		width: (data.imageAsset ?? fallbackData.imageAsset).fields.width,
+		height: (data.imageAsset ?? fallbackData.imageAsset).fields.height,
 	};
 
 	return (
-		// Render a default (basic) rich link on the server
 		<RichLink
 			richLinkIndex={richLinkIndex}
-			cardStyle="news"
-			imageData={{
-				thumbnailUrl: '',
-				altText: '',
-				width: '',
-				height: '',
-			}}
-			headlineText={element.text}
-			contentType="article"
-			url={element.url}
-			linkFormat={defaultFormat}
-			format={defaultFormat}
-			tags={[]}
-			sponsorName=""
-			isPlaceholder={true}
+			cardStyle={data.cardStyle}
+			imageData={richLinkImageData}
+			headlineText={data.headline}
+			contentType={data.contentType}
+			url={data.url}
+			starRating={data.starRating}
+			linkFormat={decideFormat(data.format)}
+			format={format}
+			tags={data.tags}
+			sponsorName={data.sponsorName}
+			contributorImage={data.contributorImage}
+			isPlaceholder={data === fallbackData}
```

Comment:
We're confident that `useApi` will return a reference to the original object, so the object comparison here is enough?

Reply:
That’s a good question, I’m not sure that this is a guarantee of SWR. I’ll switch for using checking the index is `-1`.